kyuubi-server/src/main/scala/org/apache/kyuubi/util/KubernetesUtils.scala

```diff
@@ -78,7 +78,9 @@ object KubernetesUtils extends Logging {
       .withOption(oauthTokenValue) { (token, configBuilder) =>
         configBuilder.withOauthToken(token)
       }.withOption(oauthTokenFile) { (file, configBuilder) =>
-        configBuilder.withOauthToken(Files.asCharSource(file, Charsets.UTF_8).read())
+        configBuilder.withOauthTokenProvider(new OAuthTokenProvider {
```

Comment:
seems before k8s 1.24, the default token has no expiration time, but the newer version is 1h, let's leave some comments here, including links about K8s behavior change

Reply:
done